ngraph/src/ngraph/type/element_type.cpp

```diff
@@ -325,6 +325,9 @@ size_t ngraph::compiler_byte_size(element::Type_t et)
     case element::Type_t::undefined: return 0;
     case element::Type_t::dynamic: return 0;
     }
+
+    throw ngraph_error("compiler_byte_size: Unsupported value of element::Type_t: " + 
```

Comment:
@ilyachur please take a look. I caught error from Intel compiler such as "missing return statement at end of non-void function"

Reply:
@nikita-kud Thank you! It is ok for me.